src/lambdaisland/launchpad.clj

```diff
@@ -380,9 +391,11 @@
                            (:extra-deps ctx)))
   ctx)
 
-(def before-steps [find-free-nrepl-port
+(def before-steps [
```

Comment:
Seems like an extra newline?

Reply:
yes, some emacs automation. I can remove it